independent-publisher-connectors/Star Wars/apiDefinition.swagger.json

```diff
@@ -0,0 +1,878 @@
+{

```

Comment:
@paulculmsee Please raise one PR for each connector which makes us easy to review and approve it.

Reply:
Sorry mate, did not realise I even did this :-(